lib/elixir/lib/exception.ex

```diff
@@ -653,12 +662,18 @@ defmodule FunctionClauseError do
 end
 
 defmodule Code.LoadError do
-  defexception [:file, :message]
+  defexception [file: nil, message: nil, description: "could not load file"]
```

Comment:
Sorry @eksperimental , my comment above applies only to `Code.LoadError`. Everything else is great!